portfolio/src/main/webapp/comment.js

```diff
@@ -40,32 +37,47 @@ function showMemberUI(user, loggingUrl, commentSection) {
 /**
  * Shows the UI for non-logged in guest.
  * @param {string} user User name.
- * @param {string} loggingUrl Log in or log out url.
+ * @param {string} loggingUrl Log in url.
  */
 function showGuestUI(user, loggingUrl) {
-    const loginDiv = document.createElement("div");
-    let id = document.createAttribute("id");
-    id.value = "login";
-    loginDiv.setAttributeNode(id);
-    loginDiv.innerHTML = getGreetingHTML(user, loggingUrl);
+    let htmlText = getGreetingHTML(false, user, loggingUrl);
```

Comment:
Nit: If you want to pass boolean here: /* isLoggedIn= */ false

Reply:
Fixed